src/main/java/nextstep/subway/application/LineService.java

```diff
@@ -0,0 +1,65 @@
+package nextstep.subway.application;
+
+import nextstep.subway.domain.Line;
+import nextstep.subway.domain.LineRepository;
+import nextstep.subway.domain.Station;
+import nextstep.subway.domain.StationRepository;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+import nextstep.subway.exception.LineException;
+import nextstep.subway.exception.StationException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static nextstep.subway.exception.LineExceptionMessage.NONE_EXISTS_LINE;
+import static nextstep.subway.exception.StationExceptionMessage.NONE_EXISTS_STATION;
+
+@Service
+@Transactional(readOnly = true)
+public class LineService {
+    private LineRepository lineRepository;
+    private StationRepository stationRepository;
+    public LineService(LineRepository lineRepository,StationRepository stationRepository) {
+        this.lineRepository = lineRepository;
+        this.stationRepository = stationRepository;
+    }
+
+    @Transactional
+    public LineResponse saveLine(LineRequest lineRequest) {
+        Station upStation = findStation(lineRequest.getUpStationId());
+        Station downStation = findStation(lineRequest.getDownStationId());
+        Line persistStation = lineRepository.save(lineRequest.toLine(upStation,downStation));
+        return LineResponse.of(persistStation);
+    }
+
+    public List<LineResponse> findAllLines() {
+        List<Line> lines = lineRepository.findAll();
+        return lines.stream()
+                .map(LineResponse::of)
+                .collect(Collectors.toList());
+    }
+
+    public LineResponse findLine(Long id) {
+        Line line = lineRepository.findById(id).orElseThrow(() -> new LineException(NONE_EXISTS_LINE.getMessage()));
```

Comment:
```suggestion
        Line line = lineRepository.findById(id)
                            .orElseThrow(() -> new LineException(NONE_EXISTS_LINE.getMessage()));
```
이 정도로 개행 해줘도 좋을 것 같아요..!